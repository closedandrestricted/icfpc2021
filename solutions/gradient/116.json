{"vertices":[[27,29],[54,8],[41,52],[44,78],[58,32],[104,86],[66,59],[107,57],[73,84],[48,110],[122,80],[62,87],[80,69],[22,84],[98,89],[92,78],[106,109],[127,56],[106,63],[100,63],[48,55],[81,47],[105,34],[121,78],[114,85],[112,44],[109,52],[90,77],[120,15],[117,113],[127,110],[146,39],[123,46],[92,107],[126,139],[128,70],[120,73],[127,78],[117,97],[124,84],[142,39],[146,82],[139,65],[130,103],[126,112],[108,80],[140,91],[144,89],[150,88],[168,92],[211,67],[181,118],[186,72],[138,124],[133,104],[151,119],[232,89],[198,94],[219,115],[170,89],[213,88],[193,106]]}