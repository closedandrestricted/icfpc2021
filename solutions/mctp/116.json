{"vertices":[[8,17],[7,50],[6,44],[20,66],[12,27],[23,120],[32,47],[52,122],[36,147],[47,49],[41,103],[38,74],[35,147],[61,105],[52,127],[63,54],[29,49],[30,80],[25,123],[34,108],[23,106],[38,77],[58,91],[29,83],[47,131],[23,104],[34,122],[57,100],[16,132],[23,115],[23,115],[43,155],[21,144],[31,81],[25,143],[48,88],[34,121],[54,97],[25,105],[30,168],[47,121],[37,147],[19,168],[29,91],[59,100],[25,111],[16,142],[11,161],[36,114],[16,137],[27,158],[30,112],[20,133],[11,76],[22,94],[43,154],[16,131],[46,136],[26,157],[17,130],[18,131],[10,157]]}