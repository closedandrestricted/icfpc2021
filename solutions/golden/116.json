{"vertices":[[27,28],[54,7],[42,51],[46,77],[62,30],[103,85],[66,58],[100,56],[72,84],[49,109],[102,64],[61,85],[76,73],[21,83],[97,89],[92,78],[106,109],[126,59],[101,62],[97,62],[47,54],[80,48],[112,86],[119,76],[114,81],[111,45],[108,102],[88,78],[110,15],[115,110],[122,108],[145,39],[122,46],[136,74],[124,137],[129,69],[118,73],[128,77],[116,96],[161,68],[136,36],[144,82],[137,65],[129,102],[114,46],[106,81],[138,91],[144,79],[150,88],[167,89],[211,67],[179,115],[185,71],[139,123],[132,104],[148,62],[232,89],[197,93],[218,114],[169,89],[195,97],[176,117]]}